<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>852</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>392</width>
    <height>852</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(173, 216, 230);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>392</width>
      <height>852</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page1">
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>393</width>
        <height>931</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/newPrefix/bglogin.png);</string>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>62</x>
         <y>522</y>
         <width>271</width>
         <height>48</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="forgotPassword">
       <property name="geometry">
        <rect>
         <x>132</x>
         <y>579</y>
         <width>131</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
background:none;

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="signUpButton">
       <property name="geometry">
        <rect>
         <x>239</x>
         <y>721</y>
         <width>57</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
background:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="passwordLogin">
       <property name="geometry">
        <rect>
         <x>61</x>
         <y>459</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background-color: white;
background:none;
border-radius: 10px;
padding: 10px;}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="userLogin">
       <property name="geometry">
        <rect>
         <x>62</x>
         <y>410</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background-color: white;
background:none;
border-radius: 10px;
padding: 10px;}</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>393</width>
        <height>852</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/newPrefix/bgsign.png);</string>
      </property>
      <widget class="QPushButton" name="signFinish">
       <property name="geometry">
        <rect>
         <x>61</x>
         <y>560</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="passwordSignup">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>447</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background-color: white;
background:none;
border-radius: 10px;
padding: 10px;}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="passwordConfirm">
       <property name="geometry">
        <rect>
         <x>61</x>
         <y>502</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background-color: white;
background:none;
border-radius: 10px;
padding: 10px;}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="userSignup">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>391</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background-color: white;
background:none;
border-radius: 10px;
padding: 10px;}</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="assets/login.qrc"/>
 </resources>
 <connections/>
</ui>
